components/Avatar.js

```diff
@@ -12,14 +12,17 @@ import { Flex } from './Grid';
 const getInitials = name => name.split(' ').reduce((result, value) => (result += value.slice(0, 1).toUpperCase()), '');
 
 const StyledAvatar = styled(Flex).attrs(props => ({
-  style: { backgroundImage: props.src ? `url(${props.src})` : null },
+  style: {
+    backgroundImage: props.src ? `url(${props.src})` : null,
+    backgroundSize: props.backgroundSize || 'cover',
+    backgroundColor: props.backgroundColor,
```

Comment:
@Betree : Yes, agree we are not going to use this a lot. I just added this so that I could do that yellow background for the notification icon. Here's the part where we use these props. 🐨 

https://github.com/opencollective/opencollective-frontend/blob/d0425f387334692b1caf46bdd7f6943882fa7dfb/components/TopBar.js#L246-L263

Do you think it's okay to keep it, or do you have a better idea? 🤔 🐼

Reply:
Well, this flame icon is not an Avatar, so that looks like a hack :-)

I would personally wrap all this inside a `ChangelogTrigger` (or similar) component, also to move all the logic around queries and dropdown out of the `TopBar`.

It's then easy to do something like:

```es6
export const ChangelogTrigger = () => (
  <Container borderRadius="100px" background={...}>
    {hasSeenNewUpdates ? <FlameIcon /> : <RedFlameIcon />}
    {showChangelogDropdown && (
      ...
    )}
  </Container>
)
```

Reply:
@Betree : Thanks for the idea. I've refactored that. Take a look and let me know if everything is okay now. 😄

Reply:
That looks good! Let's revert the changes to this file since they're not used anymore :wink:

Reply:
This is done. 😄